ports/starlink-ast/vcpkg.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "starlink-ast",
+  "version-string": "9.2.3",
```

```suggestion
  "version-semver": "9.2.3",
```
It would be better to use `version-semver` here. Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md